wasserstein/_wasserstein.py

```diff
@@ -28,9 +28,9 @@
 import wasserstein.config
 
 if wasserstein.config.openmp():
-	from ._wasserstein_omp import *
+	from .omp._wasserstein import *
```

Comment:
```suggestion
	from wasserstein.omp._wasserstein import *
```

Use absolute imports to make things more clear.

Same for the below, so instead of

```python
from .noomp._wasserstein import *
```

use

```python
from wasserstein.noomp._wasserstein import *
```

but GitHub's UI won't let me make that suggestion.